tests/test_conflicting_formatters.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from collections.abc import Generator
+from contextlib import contextmanager
+from pathlib import Path
+
+import pytest
+
+from pydocstringformatter import _formatting
+from pydocstringformatter import _testutils as test_utils
+from pydocstringformatter._formatting import Formatter
+from pydocstringformatter._utils import UnstableResultError
+from pydocstringformatter.run import _Run
+
+
+@contextmanager
+def patched_run(formatters: list[Formatter]) -> Generator[type[_Run], None, None]:
+    """Patches the _Run class to use only the provided formatters."""
+    old_formatters = _formatting.FORMATTERS
+    try:
```

Comment:
Why are we in a `try` here?

Reply:
To be honest, its to be safe. it makes sure the cleanup happens if there are un-handled errors while yielding the patched run (which I'm not sure it it would affect all other tests).

https://docs.python.org/3/library/contextlib.html

Reply:
Hm, for now I'd say let's remove this. If any error occurred while patching `formatters` (so while we run this test) the test (and test suite) will fail anyway. It's not necessary to guarantee a successful test suite run as if we raise we will fail anyway.

I did learn something new today though, that's always nice 😄

Reply:
Right on!
Alright, it works after removing it in [862356d](https://github.com/DanielNoord/pydocstringformatter/pull/173/commits/862356d89f3d7c39f078839963c3313c0ffb421b)